lightbluetent/templates/home/home.html

```diff
@@ -27,10 +27,21 @@ <h4 class="card-title">{{ society.name }} <small class="text-muted">{{ society.s
                 <i class="fa fa-li fa-users" title="Administrator"></i>
                 {{ society.admins|map(attribute="crsid")|join(", ") }}
             </li>
+            {% if running_meetings[society.bbb_id] %}
+            <li>
+                <i class="fa fa-li fa-video-camera"></i>
+                {{ _("Session in progress!") }}
+            </li>
+            {% endif %}
           </ul>
         </div>
         <div class="card-footer">
           <a href="{{ url_for('admin.admin', uid=society.uid) }}" class="btn btn-outline-primary">Manage</a>
+          {% if running_meetings[society.bbb_id] %}
+          <a href="{{ url_for('society.begin_session', uid=society.uid) }}" class="btn btn-outline-success">{{ _("Join session") }}</a>
```

Comment:
it is society.begin or society.begin_session?

Reply:
The route is `/<uid>/begin`, but the corresponding function is `begin_session`. Should we change the route to be `/<uid>/begin_session` for consistency?

Reply:
I think it's fine, I was just confused at first glance.